src/local/butler/reproduce.py

```diff
@@ -214,33 +218,50 @@ def _prepare_initial_environment(build_directory):
   _copy_root_subdirectory(root_dir, temp_root_dir, 'bot')
```

Comment:
would it make sense to share this functionality with the one in butler's run_bot ? Seems better to reduce the duplication here to make things more maintainable.